### PR TITLE
Refactor datetime class

### LINKED ISSUE #155: Update Class names for the Storage Package.
Update the class names in the storage package from AddressBook to PlaNus

### LINKED ISSUE #170: Update value attributes for Description and DateTime class

### PR DESCRIPTION
**Pull request by BobbyZhouZijian**
Change the value of DateTime from String to LocalDateTime.

Use a saner way to store a default DateTime object.

Fixes #170 
Fixes #155

### CODE REVIEW

File: src/main/java/seedu/address/model/task/DateTime.java [hunk 1]
```diff
@@ -30,16 +35,18 @@
...
         requireNonNull(dateTime);
         checkArgument(isValidDateTime(dateTime), MESSAGE_CONSTRAINTS);
-        value = dateTime;
+        value = LocalDateTime.parse(dateTime, FORMATTER);
+        isDefault = false;
     }
 
     /**
-     * Constructs a default {@code DateTime}.
+     * Constructs a default {@code DateTime} which is null.
```
Comment (MarcTzh) [inline src/main/java/seedu/address/model/task/DateTime.java]: why do you need to specify "which is null" when the attributes are the default values?

File: src/main/java/seedu/address/model/task/Description.java [hunk 2]
```diff
@@ -23,9 +23,10 @@
      * otherwise " " (a blank string) becomes a valid input.
      */
     public static final String VALIDATION_REGEX = "[\\p{Alnum},.!?][\\p{Alnum},.!? ]*";
-    private static final String DEFAULT_DESCRIPTION = "";
```
Comment (MarcTzh) [inline src/main/java/seedu/address/model/task/Description.java]: why cant the default desc be an empty string?

Comment (MarcTzh) [reply src/main/java/seedu/address/model/task/Description.java]: is it necessary to add an extra boolean when there was nothing wrong with the prev implementation?

File: src/main/java/seedu/address/model/task/DateTime.java (hunk 1 above)
Comment (BobbyZhouZijian) [reply src/main/java/seedu/address/model/task/DateTime.java]: sry this was a mistake. I changed it. the datetime won't be null

**Review by MarcTzh — APPROVED** [review]
LGTM

**Pull request merged**